pkg/tcl/apitcl/v1/testworkflowmetrics.go

```diff
@@ -178,10 +182,12 @@ func (s *apiTCL) sendRunWorkflowTelemetry(ctx context.Context, workflow *testwor
 	}
 
 	var dataSource string
-	if len(workflow.Spec.Content.Files) != 0 {
-		dataSource = "files"
-	} else if workflow.Spec.Content.Git != nil {
-		dataSource = "git"
+	if workflow.Spec.Content != nil {
```

Comment:
looks like this is the same code as above - could this be extracted into a shared method?

Reply:
good catch, just pushed an update